apps/website/src/components/layout/navbar/Navbar.tsx

```diff
@@ -29,13 +22,15 @@ type NavLinkProps = Omit<
     children?: React.ReactNode;
   } & React.RefAttributes<HTMLAnchorElement>;
 
-const NavLinkClasses = "block px-5 py-3 border-b-2 border-transparent hover:border-white transition-colors";
+const NavDropdownClasses = "absolute top-full right-0 z-30 mt-2 min-w-[10rem] flex flex-col rounded bg-alveus-green-900 p-2 shadow-lg";
+const NavLinkClasses = "block px-5 py-3 h-full border-b-2 border-transparent hover:border-white transition-colors";
+const NavLinkClassesActive = "border-white";
```

Comment:
As i've said for the mobile nav before, I would like to use a background color instead of the border for the dropdown items. But we can do that in next iteration :)